pjlib/src/pj/stack_win32.c

```diff
@@ -0,0 +1,142 @@
+#include <pj/os.h>
```

Comment:
Could you please add copyright and licensing info in the beginning of the files? (`stack.*` files)
You can add your name if you wish so.
(For an example, refer to the other PJSIP source files).

Reply:
copyright adde